kube_log_watcher/agents/base.py

```diff
@@ -20,11 +20,10 @@ def first_run(self):
         return True
 
     def __enter__(self):
-        self.reset()
```

Comment:
This change doesn't look safe.
How reset is executed now?

Reply:
Probably never. That is the idea of the PR. To use the agent to save its state between two executions and only add new or remove stale containers.

Reply:
Okay, so you're relying on `remove_log_target` Then it should remove items from `.logs` for scalyr and appdynamics.

Reply:
Right, remove_log_target should remove items from .logs. I missed that this was not the case. Now it removes the items.